projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -85,6 +87,40 @@ public static function build_view_data() {
 			}
 		}
 
+		if ( $is_archive ) {
```

Comment:
I'm wondering whether the `$is_archive` truly means it's an archive page as it clearly has home, archive and search pages in the following code. Maybe we could use a better name for it.

This might be out of scope, but what happens for other pages that is none of home, archive and search pages. Should we take the chance to track their views as well. In my experience, it's not rare that plugins just hook up on an URL on the front end that's not part of the routes defined in WP.

Reply:
I've tried to address your comments with my last commit but in any case let me know if there are still issues.

> what happens for other pages that is none of home, archive and search pages.

Although not able to reproduce it manually, I understand that there should be at least a fallback `other` type for the cases you mentioned.